eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -32,12 +32,15 @@ jobs:
           node common/scripts/install-run-rush.js pack --verbose $(GeneratedPackageTargets)
         displayName: "Pack libraries"
 
+      - script: |
+          node $(System.DefaultWorkingDirectory)/eng/tools/check-filter-variable.js $(packageGlobFilter)
```

Comment:
Have you considered doing this as a template parameter? You can give default values to a template parameter which if they aren't set will bet used otherwise what is passed will be used.

Reply:
Updated the solution - using the variable in globals.yml file work